api/HTMLElement.json

```diff
@@ -1506,7 +1506,8 @@
               "version_removed": "15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "11",
+              "version_removed": "14"
```

Comment:
Ah, yes, the Microdata API. This has been removed for long enough that it can be removed from BCD now. (Not in this PR though.)